sdk/core/src/policies/retry_policies/retry_policy.rs

```diff
@@ -20,6 +20,13 @@ pub trait RetryPolicy {
     fn is_expired(&self, duration_since_start: Duration, retry_count: u32) -> bool;
     /// Determine how long before the next retry should be attempted.
     fn sleep_duration(&self, retry_count: u32) -> Duration;
```

Comment:
If we have the `wait` method, do we need `sleep_duration` any more?

Reply:
It is a convenience for the simpler cases, but I don't think it is strictly required and it may not always be implemented.